src/Microsoft.AspNet.Server.Kestrel/Http/Connection.cs

```diff
@@ -75,18 +76,17 @@ private void OnRead(UvStreamHandle handle, int status, Exception error)
 
                 if (errorDone && error != null)
                 {
-                    Trace.WriteLine("Connection.OnRead " + error.ToString());
+                    KestrelTrace.Log.LogError("Connection.OnRead " + error, error);
```

Comment:
I think loggers are expected to trace the exception themselves if one is passed as an argument, so I don't think it should be necessary to concatenate the exception to the message string anymore.

Ex: https://github.com/aspnet/Logging/blob/9aa1ab4fc4e1445e1af259de1294e9328fe039ca/src/Microsoft.Framework.Logging.Console/ConsoleLogger.cs#L51